fix PVs connected to wrong places

mhier opened the pull request:
The trigger PV was connected wrongly at least if moved to an absolute position. Apparently, also
the other variables were already in the wrong place for a longer time, hence the correction of
the test.

mhier committed:
fix PVs connected to wrong places

The trigger PV was connected wrongly at least if moved to an absolute position. Apparently, also
the other variables were already in the wrong place for a longer time, hence the correction of
the test.

mhier commented:
Note that this requires the latest master of ApplicationCore to compile.

zenker commented:
Seems to work. Can you set the required version of Applicationcore in the CMakeFile.

killenb commented:
I just increased the minor version number in the ApplicationCore master. It is now on version 02.04.00.

zenker committed:
Require latest ApplicationCore version.

Introduced changes require latest version of ApplicationCore.
In addition patch version is increased.

zenker committed:
Add option to specify the tree name.

zenker committed:
Write initial values too.

zenker committed:
Fix tests.

Use modern way of connecting variables. Still don't use default
implementation of defineConnections(). This could be done using the
envelope class, but requires a config file. Here we only test the DAQ
funtionality.

zenker committed:
Merge branch 'mhier-fix-wrong-connections' of https://github.com/ChimeraTK/ApplicationCore-MicroDAQ.git into mhier-fix-wrong-connections
